recipes/bw_processing/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "bw_processing" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bw_processing-{{ version }}.tar.gz
+  sha256: f35d1e610390c4a0bf425a708dc2140be30cbbb85523da56f606e1edfa5cd567
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - fs
+    - numpy
+    - pandas
+    - python
+
+test:
+  imports:
+    - bw_processing
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/brightway-lca/bw_processing
+  summary: Tools to create structured arrays in a common format
+  license: BSD-3-Clause
+  icense_family: BSD
+  license_file:
+    - LICENSE
+  doc_url: https://3.docs.brightwaylca.org/
+  dev_url: https://github.com/brightway-lca/bw_processing
+
+extra:
+  recipe-maintainers:
+    - cmutel
```

Comment:
@cmutel, do you agree to be a maintainer of this recipe?

Reply:
Yes. Thanks for your help.